eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -1,19 +1,27 @@
 parameters:
   Directory: 'not-specified'
-  IgnoreLinksFile: "$(Build.SourcesDirectory)/eng/ignore-links.txt"
-  
+  IgnoreLinksFile: '$(Build.SourcesDirectory)/eng/ignore-links.txt'
+  WorkingDirectory: '$(System.DefaultWorkingDirectory)'
+  ScriptDirectory: 'eng/common/scripts'
+  Recursive: $false
+  Branchreplaceregex: '$(System.PullRequest.SourceRepositoryURI)'
+  Branchreplacementname: '$(System.PullRequest.SourceCommitId)'
```

Comment:
This isn't going to work. Remember why we needed to use env variables? It was because we need them to be empty if this is used outside of a PullRequest